packages/fxa-payments-server/src/components/LoadingOverlay/index.tsx

```diff
@@ -11,7 +10,11 @@ type LoadingOverlayProps = {
 
 export const LoadingOverlay = ({ isLoading }: LoadingOverlayProps) =>
   !isLoading ? null : (
-    <div id="loading-overlay" data-testid="loading-overlay">
-      <img alt="" src={spinnerImage} />
+    <div
+      id="loading-overlay"
+      className="flex justify-center items-center fixed inset-0 z-[9999]"
```

Comment:
Left out Tailwind of `background: #fbfbfe`? Do we need it?

Reply:
Nope. 👍